src/Aspire.Hosting.Python/PythonAppResourceBuilderExtensions.cs

```diff
@@ -353,6 +358,14 @@ private static IResourceBuilder<T> AddPythonAppCore<T>(
         // python will be replaced with the resolved entrypoint based on the virtualEnvironmentPath
         var resource = createResource(name, "python", Path.GetFullPath(appDirectory, builder.AppHostDirectory));
 
+        // When using the default virtual environment path, intelligently look for existing virtual environments
+        // in multiple locations: app directory first, then AppHost directory as fallback
+        var resolvedVenvPath = virtualEnvironmentPath;
+        if (virtualEnvironmentPath == DefaultVirtualEnvFolder && !Path.IsPathRooted(virtualEnvironmentPath))
```

Comment:
Why are both of these checks necessary?

`DefaultVirtualEnvFolder` is `.venv`, so if `virtualEnvironmentPath` is `.venv`, it will never be rooted.


```suggestion
        if (virtualEnvironmentPath == DefaultVirtualEnvFolder)
```

Reply:
Old deprecated methods let you specify the vdir in the AddPythonApp call.